metro.config.js

```diff
@@ -1,45 +1,75 @@
 const path = require( 'path' );
 const fs = require( 'fs' );
+const metroResolver = require( 'metro-resolver' );
```

Comment:
I'd like to share [this package](https://www.npmjs.com/package/@rnx-kit/metro-resolver-symlinks) in case it might help us on addressing the issues we have resolving the modules, especially when using `pnpm`.

**NOTE:** I did a quick check locally but I couldn't manage to make it work 😞.

Reply:
Oh nice find @fluiddot, I'll try giving that package a try too.